projects/skia/image_filter_deserialize.options

```diff
@@ -0,0 +1,4 @@
+[libfuzzer]
+max_len = 10024
+rss_limit_mb = 0
```

Comment:
this won't work on ClusterFuzz, as we cap it to 2048.

Reply:
Turns out, I don't actually need to set this anymore, I only needed to earlier when initially finding OOMs